jax/interpreters/xla.py

```diff
@@ -147,7 +147,7 @@ def primitive_computation(prim, *xla_shapes, **params):
   c.SetOpMetadata(xc.OpMetadata(op_type=prim.name, op_name=str(params)))
   backend = params.pop("backend", None)
   platform = xb.get_backend(backend).platform
-  xla_args = (_parameter_or_create_token(c, shape) for shape in xla_shapes)
+  xla_args = _map(c.ParameterWithShape, xla_shapes)
```

Comment:
Doesn't this break the token-passing ops in op-by-op mode?

Reply:
It would be good to have a test to avoid reversions!

Yes, I think you're right. Actually #1668 has the correct line here, but I forgot to copy it over when I split this PR out.

Reply:
Resolved!